### ISSUE TITLE
Burp 2020 support?

### ISSUE THREAD

**Issue by adamczi** [kacperszurek/burp_wp#5]
Hi, the extension does not work anymore with new Burp Suite. After selecting "Send to WordPress Scanner Intruder" from the context menu of a request, it actually sends it to regular Burp's Intruder. 

Tested on Burp Suite Community v2020.1, build 1297

**Comment by Hipapheralkus** [comment]
I also can't make this extender work on v2020.5. I set payload generator, but only a single payload was generated and nothing else:
![image](https://user-images.githubusercontent.com/4717664/85096008-58034500-b1f3-11ea-81c0-5df37eabc7ba.png)

**Comment by gobiasinfosec** [comment]
Yeah, its because WPScan doesn't allow them database access anymore, this plugin should be retired. https://github.com/kacperszurek/burp_wp/issues/4
